python/ray/serve/controller.py

```diff
@@ -885,18 +936,18 @@ async def broadcast_backend_config(self, backend_tag: str) -> None:
         if len(broadcast_futures) > 0:
             await asyncio.gather(*broadcast_futures)
 
-    def get_backend_config(self, backend_tag: str) -> BackendConfig:
+    def get_backend_config(self, backend_tag: BackendTag) -> BackendConfig:
         """Get the current config for the specified backend."""
-        assert (backend_tag in self.backends
+        assert (backend_tag in self.configuration_store.backends
                 ), "Backend {} is not registered.".format(backend_tag)
-        return self.backends[backend_tag].backend_config
+        return self.configuration_store.backends[backend_tag].backend_config
 
     async def shutdown(self) -> None:
         """Shuts down the serve instance completely."""
         async with self.write_lock:
-            for router in self.routers.values():
+            for router in self.actor_nursery.router_handles():
                 ray.kill(router, no_restart=True)
-            for replica_dict in self.workers.values():
+            for replica_dict in self.actor_nursery.workers.values():
```

Comment:
since we already have `actor_nursery.router_handles()` can we use something like `actor_nursery.worker_handles()`?